lld/test/ELF/debug-names-bad-offsets-sizes.s

```diff
@@ -0,0 +1,153 @@
+# REQUIRES: x86
+# RUN: llvm-mc -filetype=obj -triple=x86_64 %s -o %t1.o
+#
+# RUN: not ld.lld --debug-names %t1.o -o /dev/null 2>&1 | FileCheck %s
+#
+# CHECK: error: {{.*}}:(.debug_names): encountered unexpected end of section while reading entry
```

Comment:
We should test the filename to ensure it is correct.

You can use `FileCheck -DFILE=%t1.o` to get the filename with `[[FILE]]`. Recently, more tests have `rm -rf %t && split-file %s %t && cd %t` so that we don't need to figure out slashes/backslashes for non-Windows/Windows.

Reply:
I can successfully add "FileCheck -DFILE=%t1.o" or "FileCheck -DFILE=%t2.o to my tests.

I don't understand what you want me to do with [[FILE]]; could you be more explicit please?

I tried adding "rm -rf %t && split-file %s %t && cd %t" to some of my tests, but then  the tests fail with errors like: "split-file: error: /usr/local/google3/cmtice/LLD.llvm-project/lld/test/ELF/debug-names-parent-idx.s: no part separator was found".  I'm not sure what to do here?

Reply:
You can probably spend some time on existing tests. https://llvm.org/docs/TestingGuide.html#extra-files `#--- ` serves as a separtor.

Reply:
Just verifying that I understand what you want me to do:  For the three debug-names-*.s tests that have two input files, you want me to put both input files into the same .s file and use the #--- separator syntax (along with the split-file stuff). Is that correct?

Reply:
I've done this onw.